apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/endpoint/UtilsController.java

```diff
@@ -518,4 +523,53 @@ public Response hashByAlgorithm(
         return response.bind(dto).build();
     }
 
+    @Path("/setlog/level")
+    @POST
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_FORM_URLENCODED)
+    @Operation(
+        summary = "Set necessary logging level for specified package or logger",
+        description = "Set specified and correct LogLevel (required) to package or logger (required) with admin password (required)",
+        tags = {"utility"},
+        responses = {
+            @ApiResponse(responseCode = "200", description = "Successful execution",
+                content = @Content(mediaType = "application/json",
+                    schema = @Schema(implementation = SetLogLevelDTO.class)))
+        }
+    )
+    @RolesAllowed("admin")
+    public Response setLoggingLevel(
+        @Parameter(name = "logLevel", description = "Valid log level from available list", required = true,
+            schema = @Schema(implementation = org.slf4j.event.Level.class))
+        @FormParam("logLevel") org.slf4j.event.Level logLevel,
```

Comment:
Better to apply javax.validation on parameters, i guess.

Reply:
Added necessary